server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/UserRoutes.java

```diff
@@ -150,11 +149,18 @@ public void defineUserExist() {
     @ApiOperation(value = "Creating an user")
     @ApiImplicitParams({
             @ApiImplicitParam(required = true, dataType = "string", name = "username", paramType = "path"),
-            @ApiImplicitParam(required = true, dataTypeClass = AddUserRequest.class, paramType = "body")
+            @ApiImplicitParam(required = true, dataTypeClass = AddUserRequest.class, paramType = "body"),
+            @ApiImplicitParam(
+                    paramType = "query parameter",
+                    dataType = "String",
+                    allowEmptyValue = true,
+                    example = "?force",
+                    value = "If present, update User's password.")
```

Comment:
If present, it allows to overwrite a user password. Otherwise modification of already existing users are rejected.